prisma/migrations/20260410140000_downloadmetadata_composite_pk/migration.sql

```
@@ -0,0 +1,23 @@
+UPDATE "DownloadMetadata"
+SET "guildId" = 'UNKNOWN'
+WHERE "guildId" IS NULL;
+
+WITH ranked_rows AS (
```

SQLFluff (4.1.0)

[error] Query defines CTE "ranked_rows" but does not use it.

(ST03)